CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## v0.8.1 (2021-05-06)
   - fix SPIR-V generation from WGSL, which was broken due to "Kernel" capability
   - validate buffer storage classes
+  - fix dynamic stencil reference for Replace ops
```

Comment:
could we please move this into a separate section:
```md
## wgpu-types-0.8.1 (2021-06-08)
  - fix dynamic stencil reference for Replace ops
```

Reply:
Done